pkgs/by-name/sf/sfwbar/package.nix

```diff
@@ -1,28 +1,31 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, gtk3
-, meson
-, ninja
-, json_c
-, pkg-config
-, gtk-layer-shell
-, libpulseaudio
-, libmpdclient
-, libxkbcommon
-, alsa-lib
-, makeWrapper
-,
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gtk3,
+  meson,
+  ninja,
+  json_c,
+  pkg-config,
+  gtk-layer-shell,
+  libpulseaudio,
+  libmpdclient,
+  libxkbcommon,
+  alsa-lib,
+  makeWrapper,
 }:
-stdenv.mkDerivation rec {
-  pname = "sfwbar";
+let
   version = "1.0_beta13";
+in
+stdenv.mkDerivation {
+  pname = "sfwbar";
+  inherit version;
 
   src = fetchFromGitHub {
     owner = "LBCrion";
-    repo = pname;
+    repo = "sfwbar";
     rev = "v${version}";
-    hash = "sha256-7oiuTEqdXDReKdakJX6+HRaSi1XovM+MkHFkaFZtq64=";
+    hash = "sha256-4brP1SXaWq/L0D87rvlrWhLU1oFPSwNNxBSzRr4jsTM=";
```

Comment:
Could you try replacing this with `hash = ""` and run nix build again?

Reply:
Apparently it gives me `sha256-7oiuTEqdXDReKdakJX6+HRaSi1XovM+MkHFkaFZtq64=` as the correct hash. Dunno why before I got a different hash

Reply:
I assume that's because you have `version = "1.0_beta13";` instead of `version = "1.0_beta14";`...

Reply:
Didn't see it...

Maybe I forgot to add it while doing the editing